### DISCUSSION TITLE
How to send mp3 or mp4 file to owncast server with ffmpeg

### DISCUSSION THREAD

**Discussion by shamim-aktar-db** [owncast/owncast#1991] [Q&A]
How to send mp3 or mp4 file to owncast server with ffmpeg 

# file test.mp3 
test.mp3: MPEG ADTS, layer III,  v2.5,  24 kbps, 8 kHz, Monaural

**Comment by gabek** [comment]
You will have to output a RTMP stream (that contains video) to Owncast from whatever software you're using. You should be able to use ffmpeg to convert and stream your mp3 to RTMP and add video and send it to Owncast.
